modules/rte_mis_mail/rte_mis_mail.module

```diff
@@ -381,3 +382,138 @@ function rte_mis_mail_mail($key, &$message, $params) {
       break;
   }
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function rte_mis_mail_form_user_pass_alter(&$form, FormStateInterface $form_state, string $form_id) {
+
+  $title = $form['name']['#title'];
+  $markup = $form['mail']['#markup'];
+
+  if ($title instanceof TranslatableMarkup && $markup instanceof TranslatableMarkup) {
```

Comment:
Why we are storing it in a separate variable?

Reply:
We can remove it, did it for understanding.